cartographer/io/pcd_writing_points_processor.cc

```diff
@@ -65,6 +70,12 @@ void WriteBinaryPcdPointCoordinate(const Eigen::Vector3f& point,
   CHECK(file_writer->Write(buffer, 12));
 }
 
+void WriteBinaryPcdIntensity(const float intensity,
+                             FileWriter* const file_writer) {
+  CHECK(file_writer->Write(reinterpret_cast<const char*>(&intensity),
```

Comment:
This is undefined behavior. See https://en.cppreference.com/w/cpp/language/reinterpret_cast. Use `memcpy` as above?
nit: You probably want to `#include <cstring>` for this and use `std::memcpy`. Similar for the existing code.

Reply:
Thanks. These comments were implemented in the commit below.

Reply:
Did you see my nit? It is good practice to include what you use (IWYU) which here means: `#include <cstring>`. Also, in C++ `memcpy` is in the `std` namespace.